cmip6_downscaling/data/observations.py

```diff
@@ -1,13 +1,11 @@
 import os
```

Comment:
What do you think about renaming this file to `training.py`? After all, ERA5 is not exactly _observations_.